### DISCUSSION TITLE
Configuration for three inverters

### DISCUSSION THREAD

**Discussion by Energeticasa** [wlcrs/huawei_solar#92] [Setting up the integration] [answered]
hello,
can you explain to me how to collect the total system data?
I have three inverters, with their batteries, but when I go to insert the integration it shows me only one inverter.
![Inverter](https://user-images.githubusercontent.com/98172749/179946797-e5c52ae8-c498-4d3c-9425-8c6ed9c4f858.png)

**Comment by wlcrs** [comment, marked as answer]
When configuring the integration, you need to specify three slave id's as a comma-separated list. These were set by your installer, and cannot be detected.

You should also be able to retrieve those slave-id's by connecting with your smartphone to the Inverter Wifi AP (SUN2000-<serial number>), using the "Device commissioning" feature of the FusionSolar App and going to `Settings → Communication configuration → RS485_1`. The slave-id is mentioned in the 'COM address' field. (However, for the primary inverter it can be that the slave id must be set to '0' instead of the COM address mentioned there).

Please report back if you were able to get this working. As I only have one inverter myself, I have no way of testing this.

**Comment by Energeticasa** [comment]
Hello,
I followed the advice you gave me, took the information from the doongle and it works.
![add-on](https://user-images.githubusercontent.com/98172749/180965087-5881977d-3e4c-409d-bae9-ce8d51e722c5.png)
![add-on1](https://user-images.githubusercontent.com/98172749/180965117-bd920a6f-bd7a-4e20-a0a4-9d74cb272de8.png)
